src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx

```diff
@@ -86,15 +92,17 @@ export default class TeamSettings extends AsyncView {
               )}
             >
               <div>
-                <Link
-                  href={`/organizations/${orgId}/teams/${teamId}/remove/`}
-                  className="btn btn-danger"
+                <Confirm
+                  onConfirm={this.handleRemoveTeam}
                   priority="danger"
-                  size="small"
-                  title={t('Remove Team')}
+                  message={tct('Are you sure you want to remove the team [team]?', {
+                    team: `#${team.slug}`,
+                  })}
                 >
-                  {t('Remove Team')}
-                </Link>
+                  <Button icon="icon-trash" priority="danger" title={t('Remove Team')}>
```

Comment:
the icon color seemed a bit weird

https://www.dropbox.com/s/dp5o34nd9ef691j/Screenshot%202018-03-26%2010.31.14.png?dl=0